lecture-notes/3-bld.md

```diff
@@ -680,31 +680,271 @@ $
 
 you can use `HELLO` as a Make variable.
 
-In short, environment variables (e.g. `export CFLAGS=-g ; make hello`):
+In short, when using environment variables in commands such as `export CFLAGS=-g ; make hello`:
 
-* Tells the shell to run make with `CFLAGS` set in the environment
+* The shell runs make with `CFLAGS` set in the environment
```

Comment:
Maybe say "the variables (in this case, `CFLAGS`)" so that it's clear `CFLAGS` is part of the example?